tests/test_yaml_plugins.py

```diff
@@ -0,0 +1,22 @@
+import pytest
```

Comment:
Can we put this in tests/unit_tests/test_yaml_plugins.py ?

Reply:
Well, I moved things around and added to test_full_install.sh, let me know if that is bad.

Reply:
If we could rename that directory to something that starts with `test_*` it would work a little better. `pytest` will discover any directories that start with `test_*` and any python files named `test_*.py` within them and treat those as test modules.

How about `test_pytest`?

Reply:
Done.

Reply:
Thanks!